templates/bot/js/notification-function-base/README.md

```diff
@@ -64,6 +64,8 @@ The default notifying logic is located in `bot/src/*Trigger.*s`, and you can als
 - Use `target.type` to distinguish different targets
 - Use `target.members()`, `target.channels()` to get more targets
 - Send your own card message
+- If your notification project is based on HTTP Triggers, send POST request to the `http://<bot-endpoint>/api/notification` with your favorite tools like postman.
```

Comment:
For simplify, how about just POST "http://localhost:<port>/api/notification"?

I'd like to use `localhost` rather than `xxx.ngrok.xxx` for local debug.

Reply:
Thanks, updated

Reply:
Do we need emphasize the \<port\> as well? E.g., localhost:3978 or localhost:\<port\>?

Reply:
Oh yeah it's good to mention that, but for developers using local host they would probably know I think.